containers/unit_tests/TestWithoutInitializing.hpp

```diff
@@ -113,43 +113,51 @@ TEST(TEST_CATEGORY, resize_realloc_no_alloc_dualview) {
 }
 
 TEST(TEST_CATEGORY, resize_exec_space_dualview) {
-  using namespace Kokkos::Test::Tools;
-  listen_tool_events(Config::DisableAll(), Config::EnableFences(),
-                     Config::EnableKernels());
-  Kokkos::DualView<int*** * [1][2][3][4], TEST_EXECSPACE> bla("bla", 8, 7, 6,
-                                                              5);
-
-  auto success = validate_absence(
-      [&]() {
-        Kokkos::resize(
-            Kokkos::view_alloc(TEST_EXECSPACE{}, Kokkos::WithoutInitializing),
-            bla, 5, 6, 7, 8);
-        EXPECT_EQ(bla.template view<TEST_EXECSPACE>().label(), "bla");
-      },
-      [&](BeginFenceEvent event) {
-        if (event.descriptor().find("Kokkos::resize(View)") !=
-            std::string::npos)
-          return MatchDiagnostic{true, {"Found begin event"}};
-        return MatchDiagnostic{false};
-      },
-      [&](EndFenceEvent event) {
-        if (event.descriptor().find("Kokkos::resize(View)") !=
-            std::string::npos)
-          return MatchDiagnostic{true, {"Found end event"}};
-        return MatchDiagnostic{false};
-      },
-      [&](BeginParallelForEvent event) {
-        if (event.descriptor().find("initialization") != std::string::npos)
-          return MatchDiagnostic{true, {"Found begin event"}};
-        return MatchDiagnostic{false};
-      },
-      [&](EndParallelForEvent event) {
-        if (event.descriptor().find("initialization") != std::string::npos)
-          return MatchDiagnostic{true, {"Found end event"}};
-        return MatchDiagnostic{false};
-      });
-  ASSERT_TRUE(success);
-  listen_tool_events(Config::DisableAll());
+  constexpr bool exec_space_can_access_host_space =
+      Kokkos::SpaceAccessibility<TEST_EXECSPACE, Kokkos::HostSpace>::accessible;
+  if constexpr (!exec_space_can_access_host_space) {
+    GTEST_SKIP() << "skipping since execution space can't access HostSpace";
+  } else {
```

Comment:
Please elaborate on this change and the one below where you commented.

Reply:
In case we pass an execution space, it is used for View construction of both views. Thus, this can only work if the execution space can access `HostSpace`.